aws/sagemaker/llm-instance-recommend/lmi_recommender/lmi_config_recommender.py

```diff
@@ -32,6 +32,21 @@
     }
 }
 
+dynamic_batch_deepspeed = {
+    'smoothquant': {'model_category': {'llama-13b', 'llama-7b', 'gptneox', 'bloom', 'gptj'},
+               'instance': ['g4', 'g5', 'p4', 'p5']},
+    'strategy': {0.1: {"batch_size": 4},
```

Comment:
What are the numeric keys here? (i.e., 0.1, 0.3, ...)

Reply:
memory remaining to run inference